dataResourceExtension/dataResourceExtension_shacl.ttl

```diff
@@ -0,0 +1,93 @@
+@prefix dataResourceExtension: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/dataResourceExtension/> .
+@prefix gx: <https://registry.lab.gaia-x.eu/development/api/trusted-shape-registry/v1/shapes/jsonld/trustframework#> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+dataResourceExtension:DataResourceExtensionShape a sh:NodeShape ;
+    sh:property [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node dataResourceExtension:LinksShape ;
+            sh:order 1 ;
+            sh:path dataResourceExtension:links ],         
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node dataResourceExtension:BundleDataShape ;
+            sh:order 2 ;
+            sh:path dataResourceExtension:bundleData ];
+    sh:targetClass dataResourceExtension:DataResourceExtension .
+  
+dataResourceExtension:BundleDataShape a sh:NodeShape ;
+    sh:property [ sh:node dataResourceExtension:LinkShape ;
+            sh:description "Reference to required assets" ;
+            skos:example "for scenario, link to required hd map" ;
+            sh:name "requiredData" ;
+            sh:order 1 ;
+            sh:path dataResourceExtension:requiredData ],
+        [ sh:node dataResourceExtension:LinkShape ;
+            sh:description "Reference to optional related assets" ;
+            skos:example "at hd map, link to optional surface map" ;
+            sh:name "relatedData" ;
+            sh:order 2 ;
+            sh:path dataResourceExtension:relatedData ];
+    sh:targetClass dataResourceExtension:BundleData .    
+
+dataResourceExtension:LinksShape a sh:NodeShape ;
+    sh:property [ skos:example "link to bundle data, screenshot, video, routing, 3d preview" ;
+            sh:description "Reference to media data"@en ;
+            sh:message "Validation of media failed!"@en ;
+            sh:name "media"@en ;
+            sh:node dataResourceExtension:UrlShape ;
+            sh:order 2 ;
+            sh:path dataResourceExtension:media ],
+        [ skos:example "link to data asset, e.g. hd map" ;
+            sh:description "Reference to the data asset as url/uri of the EDC"@en ;
+            sh:message "Validation of data failed!"@en ;
+            sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:name "data"@en ;
+            sh:node dataResourceExtension:UrlShape ;
+            sh:order 1 ;
+            sh:path dataResourceExtension:data ];
```

Comment:
data muss nicht nochmal aufgeführt werden. Das passiert in Gaia-X in dem ServiceAccessPoint, siehe https://github.com/GAIA-X4PLC-AAD/gaia-x-compliant-claims-example/blob/main/src/ServiceAccessPoint-instance.json (statt EDC URL kann ich hier auch jegliche andere Adresse, unter der man das Asset findet angegeben werden)